olca-app/src/org/openlca/app/editors/processes/data_quality/DataQualityCell.java

```diff
@@ -59,12 +64,21 @@ void createComponents(Composite parent, FormToolkit toolkit) {
 		label = toolkit.createLabel(composite, text, SWT.WRAP);
 		label.addMouseTrackListener(new MouseOver());
 		label.addMouseListener(new MouseClick());
+		label.addPaintListener(new PaintListener() {
```

Comment:
Please add a short comment why this is required, why a simple `setColor` would not work here.